game.project

```diff
@@ -4,7 +4,7 @@ version = 1.0.1
 write_log = 0
 custom_resources = example/videos
 dependencies#0 = https://github.com/britzl/gooey/archive/8.1.1.zip
-dependencies#1 = https://github.com/britzl/defold-screenshot/archive/1.7.4.zip
+dependencies#1 = https://github.com/britzl/defold-screenshot/archive/1.8.1.zip
```

Comment:
This is needed for the example project to build using the updated dmScript::LuaHBuffer constructor